rules-tests/Php70/Rector/StaticCall/StaticCallOnNonStaticToInstanceCallRector/Fixture/skip_callstatic.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\Php70\Rector\StaticCall\StaticCallOnNonStaticToInstanceCallRector\Fixture;
+
+use stdClass;
+
+class Service
+{
+    public static function __callStatic($name, $arguments)
+    {
+        return Other::$name($arguments);
+    }
+}
+
+class Other
+{
+    public static function getAlbum()
+    {
+        return new stdClass;
+    }
+}
+
+class SkipCallStatic
+{
+    public function run()
+    {
+        return Service::getAlbum();
+    }
+}
```

Comment:
It's a bit unclear now what class is being refactored and which classes are here only to make the analysis work.
Could you move "source" classes that are not being refactored to the `/Source` directory?

Reply:
implemented, moved to `/Source` https://github.com/rectorphp/rector-src/pull/555/commits/cfaee2464de39527577be1e515ac5443bac8c233

Reply:
Thank you :+1: